mimic/model/nova_image_collection.py

```diff
@@ -6,7 +6,7 @@
 
 from characteristic import attributes, Attribute
 from json import dumps
-from mimic.model.rackspace_images import (ImageStore, OnMetalImage)
+from mimic.model.rackspace_images import OnMetalImage
```

Comment:
If we're re-naming this module to be specifically `nova_image_collection`, it's a little distressing to me that it is still importing rackspace-specific images.  Good that it is importing fewer rackspace-specific things though.  Do we have a ticket for cleaning this up?